### DISCUSSION TITLE
Clicking table action buttons will also select checkbox unintentionally

### DISCUSSION THREAD

**Discussion by magickaito** [Budibase/budibase#6203] [Help] [answered]
![Screenshot 2022-06-07 at 11 36 06 AM](https://user-images.githubusercontent.com/1195573/172290446-91b9eafc-2079-476c-99c8-40a82c40484b.jpg)

When clicking the action button in a table row (like the checkmark button), the row checkbox will also be selected, which feels like a bug. 
Is there a way to prevent this?

**Comment by melohagan** [comment, marked as answer]
Hi @magickaito 

I agree that does feel like a minor bug. I've raised it here: https://github.com/Budibase/budibase/issues/6209

**Reply by magickaito** [reply]
Noted will work around this at the moment

**Reply by melohagan** [reply]
A fix has been merged into master. This will go out with the next hotfix release, sometime within the next couple of weeks.
